rocketpool-cli/network/generate-tree.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 const (
-	colorReset  string = "\033[0m"
-	colorGreen  string = "\033[32m"
-	colorYellow string = "\033[33m"
+	colorReset            string = "\033[0m"
+	colorGreen            string = "\033[32m"
+	colorYellow           string = "\033[33m"
+	signallingAddressLink string = "https://docs.rocketpool.net/guides/houston/participate#setting-your-snapshot-signalling-address"
```

Comment:
nit: it's good to keep these organized

```suggestion
	colorReset  string = "\033[0m"
	colorGreen  string = "\033[32m"
	colorYellow string = "\033[33m"

	signallingAddressLink string = "https://docs.rocketpool.net/guides/houston/participate#setting-your-snapshot-signalling-address"
```